sale_order_invoicing_finished_task/tests/test_sale_order_invoicing_finished_task.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+from odoo.exceptions import ValidationError
+
+
+class TestInvoicefinishedTask(common.SavepointCase):
+
+    def setUp(self):
+        super(TestInvoicefinishedTask, self).setUp()
+        self.hour_uom = self.env.ref('uom.product_uom_hour')
+        self.env.user.company_id.project_time_mode_id = self.hour_uom.id
+        group_manager = self.env.ref('sales_team.group_sale_manager')
+        self.manager = self.env['res.users'].create({
+            'name': 'Andrew Manager',
+            'login': 'manager',
```

Comment:
```suggestion
            'login': 'test_manager',
```
Travis is failing due:

ERROR: duplicate key value violates unique constraint "res_users_login_key"
DETAIL:  Key (login)=(manager) already exists.

Reply:
Changed, thanks!